modules/dxp/apps/osb/osb-site-initializer/osb-site-initializer-customer-portal/extra/remote-app/src/routes/customer-portal/pages/Project/Overview/components/SubscriptionOverview/components/SubscriptionsNavbar/SubscriptionNavbar.js

```diff
@@ -72,32 +72,16 @@ const SubscriptionsNavbar = ({
 	setSelectedSubscriptionGroup,
 	subscriptionGroups,
 }) => {
-	const [showDropDown, setShowDropDown] = useState(false);
-	const [{isQuickLinksExpanded}] = useCustomerPortal();
-
 	const subscriptionNavbarRef = useRef();
 
+	const showDropDown = useUpdateShowDropDown();
+
 	useEffect(() => {
 		setSelectedSubscriptionGroup(subscriptionGroups[0]?.name);
 
 		// eslint-disable-next-line react-hooks/exhaustive-deps
 	}, [subscriptionGroups]);
 
-	useEffect(() => {
-		const updateShowDropDown = () => {
-			setShowDropDown(
-				subscriptionNavbarRef?.current &&
-					subscriptionNavbarRef.current.offsetWidth <
-						(isQuickLinksExpanded ? 500 : 570)
-			);
-		};
-
-		updateShowDropDown();
-		window.addEventListener('resize', updateShowDropDown);
-
-		return () => window.removeEventListener('resize', updateShowDropDown);
-	}, [isQuickLinksExpanded]);
-
 	return (
```

Comment:
I think this component can just show the three options of the menu, and we should have the components of dropdown and the navbar (rounded). What do you think?